src/menu.cpp

```diff
@@ -206,22 +206,41 @@ void init_menu()
 
 // Use non-member function, so many sources can write to the same input queue
 // Bound functions, such as member functions, can only be called, not used as pointers
-void from_isr_add_to_menu_input_queue(MENU_INPUT_t menu_input)
+void add_to_menu_input_queue(
+    MENU_INPUT_t menu_input,
+    bool from_isr)
 {
     // Write menu input GPIO pin number to queue holding all button presses
     // Remember, non-ISR queue access is not safe from within interrupts!
     // Don't care about the return value, I am ok with losing some button inputs
-    (void) xQueueSendFromISR(
+    if(from_isr)
```

Comment:
true == from_isr

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/7c45fb1afcd46a0976bd7b50a193a305785b00db